src/services/accommodation.service.js

```diff
@@ -0,0 +1,55 @@
+import models from '../models';
+
+const { Accommodation, AccommodationType } = models;
+
+/**
+ *
+ *
+ * @class AccommodationService
+ */
+class AccommodationService {
+  /**
+         *
+         *
+         * @static
+         * @param {newAccommodation} newAccommodation
+         * @returns {newAccommodation} @memberof AccommodationService
+         */
+  static async createAccommodation(newAccommodation) {
+    const { dataValues } = await Accommodation.create(newAccommodation);
+    const { updatedAt, createdAt, ...newAccommodationData } = dataValues;
+    return newAccommodationData;
+  }
+
+  /**
+   * find AccommodationType
+   * @static
+   * @param {object} property
+   * @memberof AccommodationService
+   * @returns {object} data
+   */
+  static async findAllType() {
+    const accommodationtypes = await AccommodationType.findAll();
+    const availableAccommodationTypes = accommodationtypes.map((type) => {
+      const { dataValues } = type;
+      const { updatedAt, createdAt, ...accommodationType } = dataValues;
```

Comment:
This is not how to select DB columns at the database level. There is a performance concern here.

Refer to sequelize documentation on how to select columns using the find method.

Reply:
https://github.com/andela/the_spinners-backend/pull/38/files#r364252135